Detectors/PHOS/simulation/src/DigitizerTask.cxx

```diff
@@ -53,8 +56,12 @@ InitStatus DigitizerTask::Init()
     return kERROR;
   }
 
-  // Register output container
-  mgr->RegisterAny("PHSDigit", mDigitsArray, kTRUE);
+  // Register MC Truth container
+  mLabels = new o2::dataformats::MCTruthContainer<o2::phos::MCLabel>();
+
+  // Register output containers
+  mgr->RegisterAny("PHOSDigit", mDigitsArray, kTRUE);
+  mgr->RegisterAny("PHOSDigitMCTruth", mLabels, kTRUE);
```

Comment:
As communicated privately, this IO mechanism is deprecated since we do no longer use the FairTask mechanism. 
Rather IO is typically performed by a dedicated DPL writer process. We can adjust this in another round.